tests/automount/test_automountlocation.yml

```diff
@@ -111,5 +112,6 @@
       - TestLocation_03
       - TestLocation_04
       state: absent
+      continue: yes
```

Comment:
@rjeffman In this playbook you are just ensuring TestLocation_01, TestLocation_02, TestLocation_03, TestLocation_04 are already removed, right?

Reply:
Originally, only `TestLocation_01` and `TestLocation_02` were created, and at this point, both should already be removed. `TestLocation_03` and `TestLocation_04` were never created. I'm not sure why the original author created this test in this way.

I'm not sure how to properly test `delete_continue`, so all I'm doing here is ensuring that its presence does not change the expected result of existing tests.